atat_internal_api.yaml

```diff
@@ -2038,6 +2038,37 @@ components:
   #          type: string
   #        lastName:
   #          type: string
+    CspInvocation:
+      required:
+        - method
+        - endpoint
+        - payload
+      properties:
+        method:
+          type: string
+          enum:
+            - GET
+            - POST
+            - PUT
+            - DELETE
+        headers:
+          type: object
+          additionalProperties:
```

Comment:
I was a bit confused seeing additionalProperties used this, but it checks out.
Link to docs if anyone is interested: https://swagger.io/docs/specification/data-models/dictionaries/

Reply:
Yeah - I believe this is the best practice for defining an object as a `string` -> `string` map (i.e. what we would want for HTTP Headers).

Reply:
Thanks for linking those docs